include/multigrid.h

```diff
@@ -99,6 +99,9 @@ namespace quda {
     /** Filename for where to load/store the null space */
     char filename[100];
 
+    /** Whether or not this is a staggered solve or not */
+    QudaBoolean is_staggered;
```

Comment:
There is no reason not to simply make this a `bool`: `QudaBoolean` exists only for the C interface.

Reply:
(Probably) fixed, need to go to a machine where I can compile.